notebooks/DrizzlePac/sky_matching/sky_matching.ipynb

```diff
@@ -4,14 +4,27 @@
    "cell_type": "markdown",
```

Comment:
<p>Please add comments for each section of the code for better readability. Also consider adding a <code>plt.show()</code> at the end to ensure all figures are displayed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/284/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>